src/core/decode_stream.js

```diff
@@ -134,10 +134,11 @@ class StreamsSequenceStream extends DecodeStream {
   constructor(streams, onError = null) {
```

Comment:
Can't we just remove non-stream ? (e.g. `streams = streams.filter(s => s instanceof BaseStream)` )